src/main/groovy/org/fundacionjala/gradle/plugins/enforce/utils/salesforce/OrgValidator.groovy

```diff
@@ -0,0 +1,54 @@
+package org.fundacionjala.gradle.plugins.enforce.utils.salesforce
+
+import org.fundacionjala.gradle.plugins.enforce.utils.Constants
+import org.fundacionjala.gradle.plugins.enforce.utils.Util
+import org.fundacionjala.gradle.plugins.enforce.utils.salesforce.component.validators.files.BasicOrgComponentsValidator
+import org.fundacionjala.gradle.plugins.enforce.utils.salesforce.component.validators.files.DefaultOrgComponentsValidator
+import org.fundacionjala.gradle.plugins.enforce.utils.salesforce.component.validators.files.SalesforceValidator
+import org.fundacionjala.gradle.plugins.enforce.wsc.Credential
+import org.fundacionjala.gradle.plugins.enforce.utils.salesforce.component.validators.org.OrgInterfaceValidator
+
+
+/**
+ * Validates files and folders defined in Salesforce
```

Comment:
maybe you can improve the commets. Validates files and folders defined in the user's Salesforce Org.